<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>673</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>671</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="MySlider" name="duration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="MyPushButton" name="shuffleButton">
         <property name="text">
          <string>Shuffled Off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyPushButton" name="repeatButton">
         <property name="text">
          <string>Repeat Off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="previousButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyPushButton" name="pauseButton">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyPushButton" name="muteButton">
         <property name="text">
          <string>Unmuted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volume">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="MyTreeWidget" name="tree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Library</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Playlist</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Music</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
     </property>
     <item>
      <property name="text">
       <string>Artist</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Album</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Genre</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Videos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pictures</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recorded TV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other media</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other Libraries</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>281</width>
      <height>492</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="playTab">
     <attribute name="title">
      <string>Play</string>
     </attribute>
     <widget class="MyListWidget" name="songList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>166</y>
        <width>275</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="MyLineEdit" name="currentPlaylistName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>146</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>                       Unsaved list                          </string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>currentPlaylistName</zorder>
     <zorder>songList</zorder>
    </widget>
    <widget class="QWidget" name="burnTab">
     <attribute name="title">
      <string>Burn</string>
     </attribute>
     <widget class="MyPushButton" name="sortButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>sort button</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="syncTab">
     <attribute name="title">
      <string>Sync</string>
     </attribute>
    </widget>
   </widget>
   <widget class="MyTableWidget" name="playlistTable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>511</width>
      <height>491</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>12</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>16</number>
    </attribute>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rating</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contributing artist</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Album</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Release year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Composer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Album artist</string>
     </property>
    </column>
   </widget>
   <zorder>tree</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>playlistTable</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuManage_librarier">
     <property name="title">
      <string>Manage libraries</string>
     </property>
     <addaction name="actionMusic"/>
     <addaction name="actionVideo"/>
     <addaction name="actionPictures"/>
     <addaction name="actionRecordedTV"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenURL"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionCreatePlaylist"/>
    <addaction name="actionCreateAutoPlaylist"/>
    <addaction name="actionSaveNowPlayinglist"/>
    <addaction name="actionSaveNowPlayinglistAs"/>
    <addaction name="separator"/>
    <addaction name="menuManage_librarier"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
    <addaction name="actionWorkoffline"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...                                     Ctrl + O</string>
   </property>
  </action>
  <action name="actionOpenURL">
   <property name="text">
    <string>Open URL...                             Ctrl + U</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close                                        Ctrl + W</string>
   </property>
  </action>
  <action name="actionCreatePlaylist">
   <property name="text">
    <string>Create playlist                         Ctrl + N</string>
   </property>
  </action>
  <action name="actionCreateAutoPlaylist">
   <property name="text">
    <string>Create auto playlist</string>
   </property>
  </action>
  <action name="actionSaveNowPlayinglist">
   <property name="text">
    <string>Save Now Playing list</string>
   </property>
  </action>
  <action name="actionSaveNowPlayinglistAs">
   <property name="text">
    <string>Save Now Playing list as...</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionWorkoffline">
   <property name="text">
    <string>Workoffline</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMusic">
   <property name="text">
    <string>Music</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionPictures">
   <property name="text">
    <string>Pictures</string>
   </property>
  </action>
  <action name="actionRecordedTV">
   <property name="text">
    <string>Recorded TV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">mylistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header location="global">myslider.h</header>
  </customwidget>
  <customwidget>
   <class>MyTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">mytreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">mytablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
